chain/epoch_manager/src/proposals.rs

```diff
@@ -46,8 +46,9 @@ pub fn proposals_to_epoch_info(
     validator_reward: HashMap<AccountId, Balance>,
     minted_amount: Balance,
     next_version: ProtocolVersion,
+    last_epoch_version: ProtocolVersion,
```

Comment:
nit: it feels odd to pass new, then old rather than old, then new (ie, I'd swapped the order of two params)